src/loaders/api/api.js

```diff
@@ -51,6 +51,23 @@ export function setAPI (agentIdentifier, forceDrain, runSoftNavOverSpa = false)
   var prefix = 'api-'
   var spaPrefix = prefix + 'ixn-'
 
+  /**
+   *
+   * @param {string} message
+   * @param {{[key: string]: *}} context
+   * @param {string} level
+   */
+  function log (message, context, level = 'info') {
+    handle(SUPPORTABILITY_METRIC_CHANNEL, [`API/log${level}/called`], undefined, FEATURE_NAMES.metrics, instanceEE)
+    handle('log', [now(), message, context, level], undefined, FEATURE_NAMES.logging, instanceEE)
```

Comment:
```suggestion
  function log (message, additionalProps, level = 'info') {
    handle(SUPPORTABILITY_METRIC_CHANNEL, [`API/log${level}/called`], undefined, FEATURE_NAMES.metrics, instanceEE)
    handle('log', [now(), message, additionalProps, level], undefined, FEATURE_NAMES.logging, instanceEE)
```

Reply:
These props are actually changed in the sister PR, will leave alone for now to avoid merge conflict